engine: changes to the sidebar + landing page (#972)
redpanda-data/documentation: docs/deploy/deployment-option/self-hosted/manual/index.mdx

---
title: Deploy Manually
---

<head>
    <meta name="title" content="Deploy Manually | Redpanda Docs"/>
    <meta name="description" content="Deploy Manually."/>
</head>

- [Deploy for Production](../manual/production/production-deployment)
  

>     You can use `rpm` to setup a production cluster of Redpanda using our default deployment or writing a custom deployment script.

Reviewer: ```suggestion
    You can use `rpm` to set up a production cluster of Redpanda using the default deployment or writing a custom deployment script.
```

- [Automate Deployment for Production](../manual/production/production-deployment-automation)
  

>     You can use automation tools in your environment such as Terraform and Ansible to deploy Redpanda for production.

Reviewer: ```suggestion
    You can use automation tools in your environment, such as Terraform and Ansible to deploy Redpanda for production.
```
